<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupchatwidget</class>
 <widget class="QWidget" name="groupchatwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="10,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="groupName">
         <property name="text">
          <string>群名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="msgOutput_LW"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="groupmember">
         <property name="text">
          <string>群成员</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="grouplist_LW"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="msginput_PTE"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="send_PB">
         <property name="text">
          <string>发送消息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uploadFile_PB">
         <property name="text">
          <string>上传文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
